static/docs/use-cases/huge-data-on-an-external-drive.md

```diff
@@ -0,0 +1,156 @@
+# Huge Data On An External Local Drive
+
+Sometimes the data may be huge and they are stored on an external local drive.
```

Comment:
Please start with "Some times data files may be huge and stored..."

Reply:
and update "external local drive" when a title is figured out.

Reply:
Moved to https://github.com/iterative/dvc.org/pull/565#pullrequestreview-276955425